plugins/builds/index.js

```diff
@@ -34,10 +37,11 @@ function startBuild(config) {
             return buildFactory.create({
                 jobId: job.id,
                 sha: build.sha,
-                parentBuildId: build.id,
+                parentBuildId: [build.id],
                 eventId: build.eventId,
                 username,
-                scmContext
+                scmContext,
+                start: start !== false
```

Comment:
should just be able to pass `start` since it's already a boolean

Reply:
`start` is not always passed in. So it could be `undefined`.

Reply:
oh okay 👍